baselines/compute/src/v1/addresses_client.ts.baseline

```diff
@@ -297,8 +297,8 @@ export class AddressesClient {
       request?: protos.google.cloud.compute.v1.IDeleteAddressRequest,
       options?: CallOptions):
       Promise<[
-        protos.google.cloud.compute.v1.IOperation,
-        protos.google.cloud.compute.v1.IDeleteAddressRequest|undefined, {}|undefined
+        LROperation<protos.google.cloud.compute.v1.IOperation, null>,
```

Comment:
is the metadata going to be `null` when we implement real LRO?

Reply:
I think so. MetadataType contains the status of the operation. In compute, it contains the similar status field in [operation](https://github.com/googleapis/googleapis-discovery/blob/master/google/cloud/compute/v1/compute_small.proto#L157). As that being said, we are not support `MetadataType` in diregapic LRO.
(but please correct me if I misunderstanding here) 🙏